src/Sylius/Behat/Context/Ui/Admin/ManagingCountriesContext.php

```diff
@@ -78,7 +78,7 @@ public function iShouldBeNotifiedAboutSuccess()
     }
 
     /**
-     * @Given /^(country "([^"]*)") should appear in the store$/
+     * @Then /^(country "([^"]*)") should appear in the store$/
```

Comment:
If you are already changing this line, we should get rid of nested parentheses - `(country "([^"]*)")` -> `(country "[^"]*")` or even -> `(country "[^"]+")`